site/_docs/index.md

```diff
@@ -181,3 +181,10 @@ highly welcomed!
 * *License*: [MIT](https://opensource.org/licenses/MIT)
 * Any Avatica version
 * *Maintainer*: Waylay.io
+
+### Calcite Avatica CLI: A Go-based Toool
+* [Home page](https://github.com/satyakommula96/calcite-cli)
+* Language: GO
+* *License*: [Apache 2.0](https://www.apache.org/licenses/LICENSE-2.0)
+* Any Avatica version
```

Comment:
As the CLI uses calcite-avatica-go, I think the supported versions should be `1.8.0 onwards` ans these are the versions supported by the Go driver.